core/src/test/scala/org/apache/spark/scheduler/FakeTask.scala

```diff
@@ -37,12 +37,17 @@ object FakeTask {
   }
 
   def createTaskSet(numTasks: Int, stageAttemptId: Int, prefLocs: Seq[TaskLocation]*): TaskSet = {
+    createTaskSet(numTasks, 0, stageAttemptId, prefLocs: _*)
```

Comment:
can you pass 0 as a named parameter ("stageId = 0") to make it obvious what it's being used for?